pyproject.toml

```diff
@@ -20,8 +20,9 @@ requests = "^2.30.0"
 pydantic = "^2.0.3"
 pandas = "^1.3.5"
 msgpack = "^1.0.3"
-websockets = "^10.2"
+websockets = "^11.0.3"
 sseclient-py = "^1.7.2"
+websocket = "^0.2.1"
```

Comment:
why did we add `websocket` here?

Reply:
@haxdds good point, I think I made a typo, I'll correct it in one min

Reply:
Fixed